nettests/nettests.go

```diff
@@ -90,26 +93,22 @@ func (c *Controller) Init(nt *mk.Nettest) error {
 		IncludeCountry: c.Ctx.Config.Sharing.IncludeCountry,
 		LogLevel:       "INFO",
 
-		ProbeCC:  c.Ctx.Location.CountryCode,
-		ProbeASN: fmt.Sprintf("AS%d", c.Ctx.Location.ASN),
-		ProbeIP:  c.Ctx.Location.IP,
+		ProbeCC:  c.Ctx.Session.ProbeCC(),
+		ProbeASN: c.Ctx.Session.ProbeASNString(),
+		ProbeIP:  c.Ctx.Session.ProbeIP(),
 
 		DisableReportFile: false,
 		DisableCollector:  !c.Ctx.Config.Sharing.UploadResults,
 		RandomizeInput:    false, // It's important to disable input randomization to ensure the URLs are written in sync to the DB
 		SoftwareName:      "ooniprobe-desktop",
-		SoftwareVersion:   ooni.Version,
+		SoftwareVersion:   version.Version,
```

Comment:
IRC there was something that wrote the version number using `ldflags` at build time (or at least there was an idea of doing that).

See: https://goreleaser.com/environment/#using-the-main-version

Reply:
Yeah, doing that was also my intention. Do you want me to open an issue?

Reply:
Yes let's open an issue for that.

Reply:
See https://github.com/ooni/probe-cli/issues/43